test/test_tree_2d_euler.jl

```diff
@@ -232,7 +242,8 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_vortex_amr.jl"),
       # Expected errors are exactly the same as in the parallel test!
       l2   = [2.120552206480055e-6, 0.003281541473561042, 0.003280625257336616, 0.004645872821313438],
-      linf = [4.500266027052113e-5, 0.031765399304366726, 0.03179340562764421, 0.04563622772500864])
+      linf = [4.500266027052113e-5, 0.031765399304366726, 0.03179340562764421, 0.04563622772500864],
+      coverage_override = (maxiters=10^5, tspan=(0.0, 10.5)))
```

Comment:
Why do you use a big `maxiters` here?

Reply:
There's a line `t < 0.5 * domain_length` that is only covered when running more than 10.5 seconds.